### PR TITLE
Outline

### PR DESCRIPTION
**Pull request by manwaring**
@jackwarner I plan on leaving this PR open as we work on it - I think solidifying our outline will help us get started in earnest

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        2. Utility computing
+        3. Frameworks
+            a. Serverless
+            b. Others?
+
+II.  Why should I use serverless?
+    A.  Helping you do what you know you should be doing already
+        1. API-driven development
+            a. Platforms as a first-class citizen
+            b. Meta-data analysis and informed decision making
```
Comment (jackwarner) [inline outline.md:19]: This idea is very powerful. The more I think about the implications, the bigger story I see about how server unlocks a faster trajectory up the value chain.

Comment (manwaring) [reply outline.md:19]: Maybe this is where we start out - it also seems like the topic that would be most interesting to get feedback early and often from business reps, aws reps, etc. on how they perceive the value here

File: outline.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        2. Application development
+            a. Green field
+            b. Brown field 
+        3. Automation in the cloud
+
+III.  Examples
+    A.  Monetizing your services with AWS Marketplace and API Gateway
+    B.  Secure automated serverless delivery at scale
+    C.  Voice interactions with Alexa Skills
+    D.  Function orchestration with Step Functions
```
Comment (jackwarner) [inline outline.md:40]: Step Functions are on my 'next week' TODO list. great to see them here

File: outline.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            a. Variable cost application models
+        2. Utility computing
+        3. Frameworks
+            a. Serverless
+            b. Others?
+
+II.  Why should I use serverless?
+    A.  Helping you do what you know you should be doing already
+        1. API-driven development
+            a. Platforms as a first-class citizen
```
Comment (jackwarner) [inline outline.md:18]: "Platform" here means a cloud provider (maybe stuck in recent lingo where "Platform" = "AWS"), or the idea that serverless dev should follow a platform-like approach?

File: outline.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            b. Meta-data analysis and informed decision making
+        2. Value chain analysis
+            a. Purposefully evolve the ecosystem
+            b. Focus on value creation
+        3. Two pizza teams
+            a. Decoupled architecture
```
Comment (jackwarner) [inline outline.md:24]: I've also heard 'evolutionary architecture' used as a way to describe the ability to not lock yourself in to specific tools/patterns. The GameStop example would be logging. Right now, we push data to the ELK stack, but we should be able to evolved that portion independently if they want to switch/add logging tools down the road (Splunk, log.ly, etc)

**Comment by jackwarner** [comment]
All this looks good to me. Additional thoughts in the other PR

File: outline.md (hunk 1 above)
Comment (jackwarner) [reply outline.md:19]: Agree - I think this is the big idea that, due to the advent of serverless and enterprise-ready public cloud offerings, starts to become a possibility sooner than most may expect.

**Pull request merged**